bpf/lib/nat.h

```diff
@@ -507,20 +507,23 @@ static __always_inline __maybe_unused int
 snat_v4_process(struct __ctx_buff *ctx, enum nat_dir dir,
 		const struct ipv4_nat_target *target, bool from_endpoint)
 {
-	struct icmphdr icmphdr __align_stack_8;
 	struct ipv4_nat_entry *state, tmp;
 	struct ipv4_ct_tuple tuple = {};
 	void *data, *data_end;
-	struct iphdr *ip4, iphdr;
+	struct iphdr *ip4;
 	struct {
 		__be16 sport;
 		__be16 dport;
 	} l4hdr;
 	bool icmp_echoreply = false;
 	__u64 off;
 	int ret;
+#ifndef DISABLE_ICMP_SUPPORT
```

Comment:
There is a typo for snat snat_v4_process, the right name of this is DISABLE_SNAT_ICMP_SUPPORT. The CI passed because the issue was only for ipv6. I will fix that witth my next change.